src/nvmeof/NVMeofGwMonitorClient.cc

```diff
@@ -171,8 +173,8 @@ int NVMeofGwMonitorClient::init()
     client_messenger->wait();
     return r;
   }
-  dout(10) << "nvmeof Registered monc callback" << dendl;
-
+  dout(1) << "nvmeof Registered monc callback" << dendl;
+  client.init();
```

Comment:
I'm actually not sure why you need this at all.  Client is an MDS client, seems unrelated?

Reply:
@athanatos Thank you! The nvmeof monitor client shouldn't depend on MDS. Could you please take a look at #61130?

The nvmeof gateway CI run appears green: https://github.com/baum/ceph-nvmeof/actions/runs/12392294028.